core/src/main.rs

```diff
@@ -23,8 +23,6 @@ use tauri::Manager;
 
 lazy_static! {
     pub static ref ARK_SHELF_WORKING_DIR: PathBuf = PathBuf::from(Cli::parse().path);
-    pub static ref ARK_SHELF_DATA_PATH: PathBuf =
```

Comment:
This should be reverted, see my comment in #14. Sorry, I haven't clarified about `.ark`.

I guess,  `Cli::parse().path` is what should have default value if it's not set.